docs/MapColonies/Raster/services/polygon_parts.md

```diff
@@ -0,0 +1,877 @@
+A{Get Auth Token} -->|token| B[GetCapabilities]
+    subgraph WFS
+      direction LR
+        B -->|FeatureTypeList| C{Select Feature}
+        C -->|feature| D[DescribeFeatureType]
+        D -->|schema| E{Build Your Filter}
+        E -->|filter| F[GetFeature]
+        F -->G((filtered features))
+    end
+```
+
+To list all the available feature types use the `GetCapabilities` operation and look for the `FeatureTypeList` section.
+
+<details>
+  <summary>Response</summary>
+
+```xml title="Response"
+<FeatureTypeList>
+        <FeatureType xmlns:polygon_parts="http://polygon_parts">
+            <Name>polygon_parts:bluemarble_orthophoto_polygon_parts</Name>
+            <Title>bluemarble_orthophoto</Title>
+            <Abstract/>
+            <ows:Keywords>
+                <ows:Keyword>features</ows:Keyword>
+                <ows:Keyword>polygon_parts</ows:Keyword>
+            </ows:Keywords>
+            <DefaultCRS>urn:ogc:def:crs:EPSG::4326</DefaultCRS>
+            <ows:WGS84BoundingBox>
+                <ows:LowerCorner>-180.0 -90.0</ows:LowerCorner>
+                <ows:UpperCorner>180.0 90.0</ows:UpperCorner>
+            </ows:WGS84BoundingBox>
+        </FeatureType>
+        <FeatureType xmlns:polygon_parts="http://polygon_parts">
+            <Name>polygon_parts:orthophoto_mosaic_base_orthophoto_best_polygon_parts</Name>
+            <Title>mosaic_base_orthophoto_best</Title>
+            <Abstract/>
+            <ows:Keywords>
+                <ows:Keyword>features</ows:Keyword>
+                <ows:Keyword>polygon_parts</ows:Keyword>
+            </ows:Keywords>
+            <DefaultCRS>urn:ogc:def:crs:EPSG::4326</DefaultCRS>
+            <ows:WGS84BoundingBox>
+                <ows:LowerCorner>-180.0 -90.0</ows:LowerCorner>
+                <ows:UpperCorner>180.0 90.0</ows:UpperCorner>
+            </ows:WGS84BoundingBox>
+        </FeatureType>
+    </FeatureTypeList>
+```
+
+</details>
+
+<br/>
+
+We got a `FeatureTypeList` consisting of the different `FeatureTypes` that each one hold and represent some catalog layer under the WFS service. one of them is the `polygon_parts:orthophoto_mosaic_base_orthophoto_best_polygon_parts`.`FeatureType` where polygon_parts is the namespace and the orthophoto_mosaic_base_orthophoto_best_polygon_parts is the unique `FeatureType` name under the polygon_parts namespace - it include layer's `product_name` + `product_type` + polygon_parts suffix. it is possible to query the FeatureType by its full name `polygon_parts:orthophoto_mosaic_base_orthophoto_best_polygon_parts` or short unique name `orthophoto_mosaic_base_orthophoto_best_polygon_parts`.
+The default coordinate reference system and the containing features bounding box are also presented.
+
+If you would like to view the schema of the `orthophoto_mosaic_base_orthophoto_best_polygon_parts` `FeatureType`, we could invoke the `DescribeFeatureType` request with `typeName` of `orthophoto_mosaic_base_orthophoto_best_polygon_parts`
```

Comment:
Do we have a page with the Profile details? same as here -> https://mapcolonies.github.io/developer-portal/docs/MapColonies/Raster/services/catalog/raster-catalog-profile-v1

Reply:
done